website/docs/dapp/DApp.mdx

```diff
@@ -4,127 +4,60 @@ title: DApp
-   ></iframe>
-
-:::caution
-
-Remember: NEVER share your private key publicly for production.
-
-:::
-</div>
-
-```mdx-code-block
-  </TabItem>
-  <TabItem value="issue-custom-token" label="Issue Token">
-```
-
-<div>
-  <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
-    <p>Issue a custom token on the CKB Testnet.</p>
-    <a href="./docs/dapp/issue-custom-token">Full Tutorial →</a>
-  </div>
-
-  <iframe src="https://codesandbox.io/embed/drlfr5?view=Editor+%2B+Preview&module=%2Flib.ts"
-    style={{width: "100%", height: "500px", border: "0", borderRadius: "4px", overflow:"hidden"}}
-    frameborder="0"
-    title="@offckb/xudt"
-    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
-    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
-   ></iframe>
-
-:::caution
-
-Remember: NEVER share your private key publicly for production.
+# DApps
 
-:::
-</div>
+#### Your go-to resources for dApp development on CKB.
 
-```mdx-code-block
-  </TabItem>
-  <TabItem value="create-dob" label="Create DOB">
-```
+## Examples
 
-<div>
-  <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center' }}>
-    <p>Create a on-chain digital object with Spore protocol on the CKB Testnet.</p>
-    <a href="./docs/dapp/create-dob">Full Tutorial →</a>
-  </div>
+The examples below show in code how dApps interact with CKB TestNet contracts, along with detailed tutorials explaining how to custom/build these dApps on your local CKB dev blockchain.
 
-  <iframe src="https://codesandbox.io/embed/rmwshy?view=Editor+%2B+Preview&module=%2Flib.ts"
-    style={{width: "100%", height: "500px", border: "0", borderRadius: "4px", overflow:"hidden"}}
-    frameborder="0"
-    title="@offckb/simple-transfer"
-    allow="accelerometer; ambient-light-sensor; camera; encrypted-media; geolocation; gyroscope; hid; microphone; midi; payment; usb; vr; xr-spatial-tracking"
-    sandbox="allow-forms allow-modals allow-popups allow-presentation allow-same-origin allow-scripts"
-   ></iframe>
+##### Note: Never use these address in production, or share your private key with anyone.
 
-:::caution
-
-Remember: NEVER share your private key publicly for production.
+<Tabs>
+  {tutorialFrameContent.map(
+    (
+      {
+        tabValue,
+        label,
+        tutorialTitle,
+        tutorialLink,
+        iframeSrc,
+        cautionMessage,
+      },
+      index
+    ) => (
+      <TabItem value={tabValue} label={label} default={index === 0}>
+        <TutorialFrame
+          tutorialTitle={tutorialTitle}
+          tutorialLink={tutorialLink}
+          iframeSrc={iframeSrc}
+        />
+      </TabItem>
+    )
+  )}
+</Tabs>
 
-:::
-</div>
+## Local dApp development tutorials
```

Comment:
Can you use Title case for all Headers & Subheaders?
In this case, it will be "Local DApp Development Tutorials"

Reply:
Ok